spec/lib/identity_job_log_subscriber_spec.rb

```diff
@@ -30,6 +30,26 @@
     )
   end
 
+  it 'does not log events when it is told not do' do
+    stub_request(:post, "https://llljjjj.com/").with(
+      body: "abc123",
+      headers: {
+      'Accept'=>'application/json',
+      'Accept-Encoding'=>'gzip;q=1.0,deflate;q=0.6,identity;q=0.3',
+      'Content-Type'=>'application/secevent+jwt',
+      'User-Agent'=>'Faraday v1.8.0'
+      }).to_return(status: 200, body: "", headers: {})
```

Comment:
Rubocop will probably complain about indentation and missing commas, but we probably also don't even need to stub headers at all?

```suggestion
    stub_request(:post, "https://llljjjj.com/").
      to_return(status: 200, body: "", headers: {})
```